internal_ws/yksg/src/lib.rs

```diff
@@ -345,6 +352,11 @@ impl StopgapInterpreter {
                     let size = usize::try_from(SIR.ty(&dst.ty()).size()).unwrap();
                     std::ptr::copy(ret_ptr, dst_ptr, size);
                     curframe.bbidx = bbidx;
+                } else {
+                    // The return value of `interp_step` tells the meta-tracer whether to stop or
```

Comment:
This string (or some variant of it) might better off be on the attribute in the `struct`?